### DISCUSSION TITLE
Image stored as extension .tmp when sending image as base64 from postman

### DISCUSSION THREAD

**Discussion by aymanalhattami** [spatie/laravel-medialibrary#2137] [Q&A] [answered]
I got the image sent from postman app by using addMediaFromBase64, but the image stored as .tmp extension file

$media = $market->addMediaFromBase64($input['image'])->toMediaCollection();

![spatie media library 2020-10-28 115915](https://user-images.githubusercontent.com/34315778/97418195-9505e380-1919-11eb-9bc9-5d7ecd12c097.png)

**Comment by freekmurze** [comment]
Feel free to PR a fix (including tests) that works for you

**Comment by muath-ye** [comment, marked as answer]
Are you sure the key type is a file type?
 See this image to make sure..
![image](https://user-images.githubusercontent.com/34031333/97438698-369b2e00-1936-11eb-80cd-01322841cf2e.png)
